Jenkinsfile

```diff
@@ -30,9 +35,7 @@ pipeline {
         }
         stage('Build & Test') {
             steps {
-                sh '''
-                    sbt test
-                '''
+                sh 'sbt test'
             }
         }
         stage('Publish Snapshot') {
```

Comment:
The publish step can go to the library as well, similar thing for publishing jars / docker images... so we can slim it down

Reply:
Is it always the same for all repositories? Then I can move it to the shared library.